bot/core/event.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import annotations
+
+from typing import Any
+
+import attrs
+
+BAD_ATTRS = ["app", "shard"]
```

Comment:
(see my other comment. I imagine we will get more and more of these as we handle more events)

Reply:
I don't think that's going to be the case, all hikari models have these two, but none other.